### DISCUSSION TITLE
How to implement Internationalization

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [infinitered/ignite#2405] [Q&A] [answered]
Kindly help me how can i use Internationalization in ignite template please give me any resource material 
Thankyou !

**Comment by frankcalise** [comment, marked as answer]
Hey @abdulmateentechbits! You'll find most internationalization support files in the `app/i18n` directory. Here is the quick doc page on it: https://github.com/infinitered/ignite/blob/master/docs/Internationalization.md
